sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -491,6 +492,47 @@ main().catch((error) => {
 });
 ```
 
+### Classify a document
+
+The Form Recognizer service supports custom document classifiers that can classify documents into a set of predefined categories based on a training data set. Documents can be classified with a custom classifier using the `beginClassifyDocument` method of `DocumentAnalysisClient`. Like `beginAnalyzeDocument` above, this method accepts a file or stream containing the document to be classified, and it has a `beginClassifyDocumentFromUrl` counterpart that accepts a publicly-accessible URL to a document instead.
+
+The following sample shows how to classify a document using a custom classifier:
+
+```javascript
+const { AzureKeyCredential, DocumentAnalysisClient } = require("@azure/ai-form-recognizer");
+
+async function main() {
+  const endpoint = "<endpoint>";
+  const credential = new AzureKeyCredential("<api key>");
+
+  const documentUrl =
+    "https://raw.githubusercontent.com/Azure/azure-sdk-for-js/main/sdk/formrecognizer/ai-form-recognizer/assets/invoice/Invoice_1.pdf";
+
+  const client = new DocumentAnalysisClient(endpoint, credential);
+
+  const poller = await client.beginClassifyDocumentFromUrl("<classifier id>", documentUrl);
+
+  const result = await poller.pollUntilDone();
+
+  if (result.documents === undefined || result.documents.length === 0) {
```

Comment:
Is there a reason to distinguish between the `undefined` and the empty cases?

Reply:
For document models, the service can return _undefined_ documents if document extraction is not supported in the model. It can return empty documents if document extraction is supported but no documents were extracted. In practice, using a classifier, `documents` should never be undefined. We may choose to use a stronger result type for classifiers in the future rather than reuse the `AnalyzeResult` type, but we decided that was out of scope for this first beta.